File: src/app/gkill/api/req_res/update_nlog_response.go
package req_res

import (
	"github.com/mt3hr/gkill/src/app/gkill/api/message"
	"github.com/mt3hr/gkill/src/app/gkill/dao/reps"
)

type UpdateNlogResponse struct {
	Messages []*message.GkillMessage `json:"messages"`

	Errors []*message.GkillError `json:"errors"`

	UpdatedNlog *reps.Nlog `json:"updated_nlog"`

	UpdatedNlogKyou *reps.Kyou `json:"updated_nlog_kyou"`
}

File: src/classes/api/req_res/update-nlog-response.ts
'use strict';

import { Nlog } from '@/classes/datas/nlog';
import { GkillAPIResponse } from '../gkill-api-response';
import { Kyou } from '@/classes/datas/kyou';


export class UpdateNlogResponse extends GkillAPIResponse {


    updated_nlog: Nlog;

    updated_nlog_kyou: Kyou;

    constructor() {
        super()
        this.updated_nlog = new Nlog()
        this.updated_nlog_kyou = new Kyou()
    }


}



